### PR TITLE
Add configurable APK installation timeout.

### PR DESCRIPTION
**Pull request by koral--**
https://github.com/gojuno/commander/issues/7 continuation.

### CODE REVIEW

File: composer/src/main/kotlin/com/gojuno/composer/Args.kt [hunk 1]
```diff
@@ -96,7 +97,15 @@ data class Args(
...
-        var devicePattern: String = ""
+        var devicePattern: String = "",
+
+        @Parameter(
+                names = arrayOf("--install-timeout"),
+                required = false,
+                description = "APK installation timeout in seconds. If not passed defaults to 120 seconds (2 minutes). Applicable to both test APK and APK under test.",
+                order = 11
+        )
+        var installTimeoutSeconds: Int = TimeUnit.MINUTES.toSeconds(2).toInt()
```
Comment (artem-zinnatullin) [inline composer/src/main/kotlin/com/gojuno/composer/Args.kt]: Not sure this is how you set default value, most likely JCommander will override that anyway

Can you add a test please?

File: composer/src/test/kotlin/com/gojuno/composer/ArgsSpec.kt [hunk 2]
```diff
@@ -144,4 +144,15 @@ class ArgsSpec : Spek({
...
         }
     }
+
+    context("parse args with passed --install-timeout") {
+
+        val args by memoized {
+            parseArgs(rawArgsWithOnlyRequiredFields + arrayOf("--install-timeout", "600"))
+        }
+
+        it("parses --install-timeout correctly") {
```
Comment (artem-zinnatullin) [inline composer/src/test/kotlin/com/gojuno/composer/ArgsSpec.kt]: But what about default value? :)

**Comment by artem-zinnatullin** [comment]
@koral-- thanks!

Pls report if it solved your issue :)

**Comment by koral--** [comment]
OK, thanks for the quick response.
I'll create appropriate PR in https://github.com/trevjonez/composer-gradle-plugin and check it ASAP.

**Comment by artem-zinnatullin** [comment]
Oh, so you use Gradle Plugin, patching 3 repositories is not so fun I
guess, glad we have fast and easy release process :)

**Comment by trevjonez** [comment]
I have an issue open for it there already. https://github.com/trevjonez/composer-gradle-plugin/issues/8

Shoot me a pr or I can get it in and updated first thing in the morning.

**Comment by trevjonez** [comment]
this repo is in need of a readme update to reflect this new flag.

**Comment by koral--** [comment]
Just added README update in #110.
I'll try to create PR in plugin repo ASAP.

**Comment by yunikkk** [comment]
Oh, and other flags, I guess. Will add them separately.

**Comment by trevjonez** [comment]
pr is up for this. any review is welcome. thus far the gradle plugin has been a solo effort so I will merge in an hour or so if nothing moves on it

**Comment by koral--** [comment]
@artem-zinnatullin 
> Pls report if it solved your issue :)

I would said that it is partially solved. Indeed installation time is not an issue now. So change from this PR is OK.

But it seems that installation hangs when output file reaches 64KB. I'll investigate that further and create separate issue possibly.

**Comment by artem-zinnatullin** [comment]
@koral-- yeah, what a "wonderful" world of programmatic binary invocations…

**Pull request merged**